bin/common.bash

```diff
@@ -351,6 +351,13 @@ validate_config() {
             fi
         done
 
+        schema_validation_result="$(mktemp  --suffix='txt')"
```

Comment:
Could you also add a cli, `./bin/apps validate sc/wc`, command to run the validator on sc/wc config? :slightly_smiling_face: